app/views/foreman/unattended/userdata-katello.erb

```diff
@@ -16,6 +16,9 @@ manage_etc_hosts: true
 ssh_authorized_keys:
```

Comment:
@stbenjam can you explain about more about this comment?  Is this a problem or a general observation?